app/components/primer/beta/auto_complete/item.rb

```diff
@@ -29,14 +30,17 @@ def initialize(value:, selected: false, disabled: false, **system_arguments)
           @system_arguments[:"aria-disabled"] = true if disabled
 
           @system_arguments[:classes] = class_names(
-            "autocomplete-item",
+            "ActionList-item",
             system_arguments[:classes],
-            "disabled" => disabled
           )
         end
 
         def call
-          render(Primer::BaseComponent.new(**@system_arguments)) { content }
+          render(Primer::BaseComponent.new(**@system_arguments)) do |_c|
+            render(Primer::BaseComponent.new(tag: :span, classes: "ActionList-content")) do |_c|
+              render(Primer::BaseComponent.new(tag: :span, classes: "ActionList-item-label")) { content }
+            end
+          end
```

Comment:
It should be possible to leave off the block argument, i.e. the `|_c|` part.

```suggestion
          render(Primer::BaseComponent.new(**@system_arguments)) do
            render(Primer::BaseComponent.new(tag: :span, classes: "ActionList-content")) do
              render(Primer::BaseComponent.new(tag: :span, classes: "ActionList-item-label")) { content }
            end
          end
```